Makefile

```diff
@@ -270,13 +268,13 @@ prd/lib/: src/lib/d3-3.3.1.min.js \
 prd/lib/build.js: src/lib/jquery-2.0.3.min.js \
 		    src/lib/bootstrap-3.3.1.min.js \
 		    src/lib/moment-with-customlocales.min.js \
-		    src/lib/typeahead-0.9.3.min.js src/lib/angular.min.js \
+		    src/lib/typeahead-0.9.3.min.js src/lib/angular.js \
```

Comment:
Why did you use the uncompressed angular ?  
Angular doesn't  seems to be compressed in the final build.js file.